src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -195,24 +195,7 @@ public ListCmd()
         @Override
         protected void execute(CommandEvent event) 
         {
-            if(!bot.getPlaylistLoader().folderExists())
-                bot.getPlaylistLoader().createFolder();
-            if(!bot.getPlaylistLoader().folderExists())
-            {
-                event.reply(event.getClient().getWarning()+" Playlists folder does not exist and could not be created!");
-                return;
-            }
-            List<String> list = bot.getPlaylistLoader().getPlaylistNames();
-            if(list==null)
-                event.reply(event.getClient().getError()+" Failed to load available playlists!");
-            else if(list.isEmpty())
-                event.reply(event.getClient().getWarning()+" There are no playlists in the Playlists folder!");
-            else
-            {
-                StringBuilder builder = new StringBuilder(event.getClient().getSuccess()+" Available playlists:\n");
-                list.forEach(str -> builder.append("`").append(str).append("` "));
-                event.reply(builder.toString());
-            }
+            new PlaylistsCmd(bot).replyList(event, false);
```

Comment:
Do not instantiate a new instance of a command every time. Make the `replyList` method static & update the arguments to include the playlist loader.
```suggestion
            PlaylistsCmd.replyList(event, bot.getPlaylistLoader(), false);
```